2022-01-15 15:49:18,108 INFO    Thread-12 :24105 [wandb_setup.py:_flush():71] setting env: {'project': 'Amazon-Reviews', 'entity': 'ewebid', 'root_dir': '/Users/ewelina/Desktop/MLOps_Amazon_review_project', 'run_id': 'ecqi2hq1', 'sweep_param_path': '/Users/ewelina/Desktop/MLOps_Amazon_review_project/wandb/sweep-ncwmt14p/config-ecqi2hq1.yaml', 'sweep_id': 'ncwmt14p'}
2022-01-15 15:49:18,108 INFO    Thread-12 :24105 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-15 15:49:18,109 INFO    Thread-12 :24105 [wandb_init.py:_log_setup():371] Logging user logs to /Users/ewelina/Desktop/MLOps_Amazon_review_project/wandb/run-20220115_154918-ecqi2hq1/logs/debug.log
2022-01-15 15:49:18,110 INFO    Thread-12 :24105 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/ewelina/Desktop/MLOps_Amazon_review_project/wandb/run-20220115_154918-ecqi2hq1/logs/debug-internal.log
2022-01-15 15:49:18,111 INFO    Thread-12 :24105 [wandb_init.py:init():404] calling init triggers
2022-01-15 15:49:18,111 INFO    Thread-12 :24105 [wandb_init.py:init():409] wandb.init called with sweep_config: {'batch_size': 250, 'drop_out': 0.35, 'epochs': 5, 'lr': 0.01, 'optimizer': 'adam'}
config: {}
2022-01-15 15:49:18,111 INFO    Thread-12 :24105 [wandb_init.py:init():460] starting backend
2022-01-15 15:49:18,111 INFO    Thread-12 :24105 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-15 15:49:18,117 INFO    Thread-12 :24105 [backend.py:ensure_launched():216] starting backend process...
2022-01-15 15:49:18,126 INFO    Thread-12 :24105 [backend.py:ensure_launched():221] started backend process with pid: 24207
2022-01-15 15:49:18,130 INFO    Thread-12 :24105 [wandb_init.py:init():469] backend started and connected
2022-01-15 15:49:18,138 INFO    Thread-12 :24105 [wandb_run.py:_config_callback():956] config_cb None None {'batch_size': 250, 'drop_out': 0.35, 'epochs': 5, 'lr': 0.01, 'optimizer': 'adam'}
2022-01-15 15:49:18,147 INFO    Thread-12 :24105 [wandb_init.py:init():533] updated telemetry
2022-01-15 15:49:18,187 INFO    Thread-12 :24105 [wandb_init.py:init():563] communicating current version
2022-01-15 15:49:23,005 INFO    Thread-12 :24105 [wandb_init.py:init():568] got version response 
2022-01-15 15:49:23,006 INFO    Thread-12 :24105 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-15 15:49:23,353 INFO    Thread-12 :24105 [wandb_init.py:init():606] starting run threads in backend
2022-01-15 15:49:28,366 INFO    Thread-12 :24105 [wandb_run.py:_console_start():1810] atexit reg
2022-01-15 15:49:28,367 INFO    Thread-12 :24105 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-15 15:49:28,367 INFO    Thread-12 :24105 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-15 15:49:28,368 ERROR   Thread-12 :24105 [wandb_run.py:_redirect():1748] Failed to redirect.
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1741, in _redirect
    out_redir.install()
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/opt/anaconda3/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-01-15 15:49:28,368 INFO    Thread-12 :24105 [wandb_init.py:init():633] run started, returning control to user process
2022-01-15 15:49:33,648 INFO    Thread-12 :24105 [wandb_run.py:finish():1457] finishing run ewebid/Amazon-Reviews/ecqi2hq1
2022-01-15 15:49:33,649 INFO    Thread-12 :24105 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-15 15:49:33,658 INFO    Thread-12 :24105 [wandb_run.py:_restore():1752] restore
2022-01-15 15:49:33,669 INFO    Thread-12 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 772
  total_bytes: 772
}

2022-01-15 15:49:34,070 INFO    Thread-12 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 772
  total_bytes: 772
}

2022-01-15 15:49:34,398 INFO    Thread-12 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 772
  total_bytes: 6917
}

2022-01-15 15:49:34,501 INFO    Thread-12 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 772
  total_bytes: 18072
}

2022-01-15 15:49:34,608 INFO    Thread-12 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 772
  total_bytes: 18072
}

2022-01-15 15:49:34,724 INFO    Thread-12 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 772
  total_bytes: 18072
}

2022-01-15 15:49:34,829 INFO    Thread-12 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 772
  total_bytes: 18072
}

2022-01-15 15:49:34,935 INFO    Thread-12 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18072
  total_bytes: 18072
}

2022-01-15 15:49:35,042 INFO    Thread-12 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18072
  total_bytes: 18072
}

2022-01-15 15:49:35,149 INFO    Thread-12 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18072
  total_bytes: 18072
}

2022-01-15 15:49:35,258 INFO    Thread-12 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18072
  total_bytes: 18072
}

2022-01-15 15:49:35,365 INFO    Thread-12 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18072
  total_bytes: 18072
}

2022-01-15 15:49:35,470 INFO    Thread-12 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18072
  total_bytes: 18072
}

2022-01-15 15:49:36,046 INFO    Thread-12 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18072
  total_bytes: 18072
}

2022-01-15 15:49:36,801 INFO    Thread-12 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18072
  total_bytes: 18072
}
local_info {
}

2022-01-15 15:49:38,048 INFO    Thread-12 :24105 [wandb_run.py:_append_files():2180] logging synced files
